src/main/java/seedu/address/model/concert/ConcertContact.java

```diff
@@ -0,0 +1,86 @@
+package seedu.address.model.concert;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+
+import seedu.address.model.person.Person;
+
+/**
+ * Represent association between a person and concert in the address book
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class ConcertContact {
+    private static final Map<Concert, ArrayList<ConcertContact>> concertContacts = new HashMap<>();
+    private final Person person;
+    private final Concert concert;
+
+    /**
+     * Constructs Concert Contact
+     *
+     * @param person
+     * @param concert
+     */
+    public ConcertContact(Person person, Concert concert) {
+        requireAllNonNull(person, concert);
+
+        this.person = person;
+        this.concert = concert;
+
+        addContactToConcert(this); //instantly adds Person to Concert list as ConcertContact created
+    }
+
+    public Person getPerson() {
+        return this.person;
+    }
+    public Concert getConcert() {
+        return this.concert;
+    }
+
+    /**
+     * Adds ConcertContact to Concert list in the map, based on concert linked to Person
+     *
+     * @param contact
+     */
+    private static void addContactToConcert(ConcertContact contact) {
+        concertContacts.computeIfAbsent(contact.getConcert(), key -> new ArrayList<>()).add(contact);
+    }
+
+    /**
+     * Returns the list of ConcertContacts in the Concert
+     *
+     * @return List of ConcertContact
+     */
+    public ArrayList<ConcertContact> concertContactList(Concert concert) {
```

Comment:
this should be named: getConcertContactList or something since it is a getter, also should return an immutable object instead of the array list directly